spec/features/restaurant/user_sees_details_of_restaurant_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature 'user views details of a restaurant', %{
+  As a user
+  I want to click on a restaurant name
+  so I can see its details
+} do
+
+  # [x] - I can click on a restaurant's name and get redirected
+  # [x] - I am redirected to the restaurant show page
+  # [x] - I can see the details of the restaurant
+
+  restaurant =  FactoryGirl.create(:restaurant)
```

Comment:
Don't use normal assignment here - use a `let` statement otherwise your db won't be cleaned properly